frontend/src/components/Pages/FrontPage/MissionOverview/MissionStatusDisplay.tsx

```diff
@@ -42,6 +43,16 @@ export function displayIcon(status: MissionStatus) {
 }
 
 export function MissionStatusDisplay({ status }: StatusProps) {
+    const { TranslateText } = useLanguageContext()
+    return (
+        <StyledStatusIcon>
+            {displayIcon(status)}
+            <Typography>{TranslateText(status)}</Typography>
+        </StyledStatusIcon>
+    )
+}
+
+export function MissionStatusDisplayWithHeader({ status }: StatusProps) {
     const { TranslateText } = useLanguageContext()
     return (
         <StyledStatusDisplay>
```

Comment:
Good idea making a new function for the version with header! You can call the MissionStatusDisplay in this function to avoid duplicate code :)